scheduler-interface-spec.md

```diff
@@ -924,6 +927,13 @@ message EventRecord {
      ALLOC_TIMEOUT      = 502;  // Allocation cancelled due to timeout
      ALLOC_REPLACED     = 503;  // Allocation replacement (placeholder)
      ALLOC_NODEREMOVED  = 504;  // Allocation cancelled, node removal
+
+     UG_USER_LIMITSET   = 600;  // Limit is configured for a given user
+     UG_GROUP_LIMITSET  = 601;  // Limit is configured for a given group
+     UG_APP_LINK        = 602;  // Linkage is created between an application and a group
+     UG_APP_UNLINK      = 603;  // Unlinking an application and a group
+     UG_USER            = 604;  // ObjectID references a user
+     UG_GROUP           = 605;  // ObjectID references a group
```

Comment:
When incrementing/decrementing resources, there will be separate event for the user and a group. It tells if the ObjectID refers to a user or group.